redbeams/src/main/java/com/sequenceiq/redbeams/service/stack/RedbeamsStartService.java

```diff
@@ -41,7 +41,7 @@ public void startDatabaseServer(String crn) {
 
         dbStackStatusUpdater.updateStatus(dbStack.getId(), DetailedDBStackStatus.START_REQUESTED);
 
-        flowManager.notify(RedbeamsStartEvent.REDBEAMS_START_EVENT.selector(),
-                new RedbeamsEvent(RedbeamsStartEvent.REDBEAMS_START_EVENT.selector(), dbStack.getId()));
+        flowManager.notify(RedbeamsStartEvent.REDBEAMS_CERT_REPLACE_EVENT.selector(),
+                new RedbeamsEvent(RedbeamsStartEvent.REDBEAMS_CERT_REPLACE_EVENT.selector(), dbStack.getId()));
```

Comment:
Please revert this class as per above; let us keep `REDBEAMS_START_EVENT` as the init event.